utils/connection.py

```
@@ -0,0 +1,73 @@
+    correct_time = datetime.datetime.now()
+    target_date = correct_time.strftime('%Y-%m-%d')
+    return target_date
+
+# Database에 연걸하는 메소드
+# Parameter : String sql = 불러오고자 하는 Select문 
+# Return : 반환 받은 DataSet을 Dictonary로 파싱 후 반환
+
+def selectDB(sql):
+    if os.getenv("GITHUB_ACTIONS") != "true":
+        load_dotenv()
+        
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)